.taskcluster.yml

```diff
@@ -37,8 +37,12 @@ tasks:
            # Disabled due to legitimate failures.
            # See https://github.com/mozilla-releng/scriptworker/issues/415
            ['py38', 'python:3.8', {NO_TESTS_OVER_WIRE: '1'}, []],
-           ['py37-cot', 'python:3.7', {NO_CREDENTIALS_TESTS: '1'}, []],
-           ['py38-cot', 'python:3.8', {NO_CREDENTIALS_TESTS: '1'}, []],
+           ['py37-cot', 'python:3.7', {
+                NO_CREDENTIALS_TESTS: '1'
+            }, ['secrets:get:repo:github.com/mozilla-releng/scriptworker:github']],
+           ['py38-cot', 'python:3.8', {
+                NO_CREDENTIALS_TESTS: '1',
+            }, ['secrets:get:repo:github.com/mozilla-releng/scriptworker:github']],
```

Comment:
Hm. Now that we have secrets, we could add integration tests.
This is a comment for future enhancements (for anyone to do, not necessarily you), and definitely not a blocker for this PR :)